### ISSUE TITLE
issue grabbing keyboard from certain applications

### ISSUE THREAD

**Issue by porcupinecoder** [Bugswriter/notflix#42]
on kde wayland

run script, bar shows up but cannot get it to take keyboard focus from things like kde's integrated run/search tool or my alactritty/fish shell. with applications like chrome and steam it is no issue

**Comment by Jitendra300** [comment]
in debian 11, wayland too it's not working, idk why but it doesn't grab my keyboard....
